README.md

```diff
@@ -513,12 +513,13 @@ sections = {
   lualine_a = {
     {
       'diff',
-      colored = true, -- Displays a colored diff status if set to true
+      colored = true,          -- Displays a colored diff status if set to true.
+      show_all_if_any = false, -- Shows every value if any is non-zero.
```

Comment:
I think we should try another name . This one seems confusing . But I'm not sure what it should be named either.